SigningService/StrongNameSignerService.cs

```diff
@@ -0,0 +1,33 @@
+﻿using SigningService.Agents;
```

Comment:
This should not exist in the root project directory. Lets move it into \Signers\StrongName\